transmission_interface/src/simple_transmission_loader.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright 2022 PAL Robotics S.L.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "transmission_interface/simple_transmission_loader.hpp"
```

Comment:
shouldn't this include be in the same group with the rest of the local ones?

Reply:
Iirc it should be at the top, otherwise the code formatter won't be happy since it doesn't follow the preferred order, which is: "own_header.h" -> external C headers -> external C++ headers -> other headers.

I'll double check it though

Reply:
ok, that's alright